zenodo/modules/deposit/static/json/zenodo_deposit/deposit_form.json

```
@@ -535,6 +543,10 @@
                     "value": "10.13039/501100002428",
                     "name": "Austrian Science Fund (AT)"
                   },
+                  {
+                    "value": "10.13039/501100000024",
+                    "name": "Canadian Institutes of Health Research (EN)"
```

```suggestion
                    "name": "Canadian Institutes of Health Research (CA)"
```

Hm, these guys are Canadian, not sure why `CA` is not in the CrossRef metadata response... Anyways, same for "Natural Sciences and Engineering Research Council of Canada" and the "Social Science Research Council" (US).